server/src/main/java/com/paypal/selion/proxy/SeLionRemoteProxy.java

```diff
@@ -119,8 +145,12 @@ public SeLionRemoteProxy(RegistrationRequest request, Registry registry) throws
         info.append("New proxy instantiated for the machine ").append(machine);
         proxyLogger.info(info.toString());
         info = new StringBuffer();
-        info.append("SeLionRemoteProxy will attempt to recycle the node [");
-        info.append(machine).append("] after ").append(getMaxSessionsAllowed()).append(" unique sessions");
+        if (isEnabledMaxUniqueSessions()) {
+            info.append("SeLionRemoteProxy will attempt to recycle the node [");
+            info.append(machine).append("] after ").append(getMaxSessionsAllowed()).append(" unique sessions");
+        } else {
+            info.append("SeLionRemoteProxy will not attempt to recycle the node based on unique session counting.");
```

Comment:
I probably should add the node `machine` id in this message, to be consistent with the message in the `true` case.